src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -61,7 +61,8 @@
                   i18n>Bucket names cannot be formatted as IP address.</span>
             <span class="invalid-feedback"
                   *ngIf="bucketForm.showError('bid', frm, 'onlyLowerCaseAndNumbers')"
-                  i18n>Bucket names can only contain lowercase letters, numbers, and hyphens.</span>
+                  i18n>Bucket names can only contain lowercase letters, numbers, and hyphens.
+                       Bucket names should not end in dot (.)</span>
```

Comment:
If they can only contain "lowercase letters, numbers, and hyphens" then it's implicit that they cannot end in dot. right?

Reply:
Yes. But let's say the user is entering a bucket name `test.bucket`. As soon as he types `test.`, this error message will be showing `Bucket names can only contain lowercase letters, numbers, and hyphens.` since the last character was a dot(.). I thought it might cause a confusion to the user. That's why I added the 'extra' message at the end.

Reply:
IMHO it's unnecessary.

Reply:
Removed it.